### ISSUE TITLE
reduce time for obtaining IPv6 router advertisment after rebooting

### ISSUE THREAD

**Issue by stapelberg** [rtr7/router7#12]
Currently, after an update, it sometimes takes up to a minute for the router to become healthy:

```
2018/10/24 08:35:53 unhealthy after 1m17.275630943s: unhealthy: ping6gw/<default-gateway>: i/o timeout
2018/10/24 08:35:54 became healthy after 1m18.332338502s
```

I think this is because we don’t solicit router advertisments, we just listen for them.

We could probably make `dhcp6` send a router solicitation packet once it can successfully bind to the `uplink0` interface.

**Comment by stapelberg** [comment]
This will be fixed with commit https://github.com/golang/go/commit/9fc22d29092933460fe00bdaccea179f29e9960d, which will be included in Go 1.12.

**Comment by stapelberg** [comment]
Closing this now that Go 1.12 has been released for a while.
